Author: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>

The PowerVR GL implementation used in Tizen Mobile 2.1 installs libGLESv2.so.1,
not .2. Adjust the library being looked for, otherwise the detection will fail.

Not upstreamable.
--- src/ui/gl/gl_implementation_x11.cc
+++ src/ui/gl/gl_implementation_x11.cc
@@ -139,7 +139,7 @@ bool InitializeGLBindings(GLImplementation implementation) {
       break;
     }
     case kGLImplementationEGLGLES2: {
-      base::NativeLibrary gles_library = LoadLibrary("libGLESv2.so.2");
+      base::NativeLibrary gles_library = LoadLibrary("libGLESv2.so.1");
       if (!gles_library)
         return false;
       base::NativeLibrary egl_library = LoadLibrary("libEGL.so.1");